arangod/Cluster/ClusterInfo.cpp

```diff
@@ -6426,7 +6424,7 @@ ClusterInfo::SyncerThread::SyncerThread(Server& server,
 
 ClusterInfo::SyncerThread::~SyncerThread() { shutdown(); }
 
-bool ClusterInfo::SyncerThread::notify(velocypack::Slice const& slice) {
+bool ClusterInfo::SyncerThread::notify(velocypack::Slice const&) {
```

Comment:
Looks like we can change the signature here, in the header and at the call site.
```suggestion
bool ClusterInfo::SyncerThread::notify() {
```

Reply:
Ok